src/db/schemas/DND5/DND5Spell.ts

```diff
@@ -59,6 +59,7 @@ export class DND5Spell extends Model<DND5Spell> {
     materials: string;
 
     @AllowNull(true)
+    @Default(0)
```

Comment:
Huh? Erklären bitte <3

Reply:
Weiß ich auch nicht. Kann ich wegnehmen. Ich mag null Werte nicht...Soll ich das wegnehmen?

Reply:
Wenn du etwas tust und nicht weißt warum, dann tu es nicht! Und ne änderung in der DB zu machen nur weil du etwas nicht magst is nicht gut.